docker/compose/docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '2'
+
+services:
+  radicale:
+    image: kozea/radicale:2.1.8
+    ports:
+      - "5232:5232"
+    volumes:
+      - ./config:/etc/radicale
```

Comment:
You probably also want to mount the data?

Reply:
actually, no. for an evualuation setup that suffices imo.